articles/native-platforms/ionic/06-authorization.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Authorization
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to access secure content in the app.
+---
+
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '06-Authorization',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/docs/quickstart/native/ionic/05-rules' }) %>
```

Comment:
@lobo Throws `Error: Could not find include ../_includes/_authorization-introduction`. There is no such directory.